src/http.rs

```diff
@@ -52,6 +52,7 @@ impl HttpError {
 }
 
 pub type HttpClientFuture = <HttpClient as Service<http::Request<Body>>>::Future;
+pub type BodyBox = http_body::combinators::BoxBody<hyper::body::Bytes, hyper::Error>;
```

Comment:
Is `pub type` necessary here?

Reply:
Bruce: No, because. pub is only necessary if used the listed symbols somewhere else. Marks it exported for the rest of vector.

With this pub type BodyBox we can then use it in all the other modules